### PR TITLE
Translated all  overall descriptions in README

### LINKED ISSUE #18308: [translation]: Update Russian translations for contributing documentations
#### Update the following in Russian from their English version available in [`/docs`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs) directory:

- [x] [`/docs/russian/CONTRIBUTING.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/russian/CONTRIBUTING.md) from its english version at [`/CONTRIBUTING.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/CONTRIBUTING.md)

- [ ] [`/docs/russian/README.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/russian/README.md) from its english version at [`/docs/README.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/README.md)

- [ ] [`/docs/russian/how-to-catch-outgoing-emails-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/russian/how-to-catch-outgoing-emails-locally.md) from its english version at [`/docs/how-to-catch-outgoing-emails-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-catch-outgoing-emails-locally.md)

- [x] [`/docs/russian/how-to-setup-freecodecamp-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/russian/how-to-setup-freecodecamp-locally.md) from its english version at [`/docs/how-to-setup-freecodecamp-locally.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-setup-freecodecamp-locally.md)

- [x] [`/docs/russian/how-to-work-on-coding-challenges.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/russian/how-to-work-on-coding-challenges.md) from its english version at [`/docs/how-to-work-on-coding-challenges.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-work-on-coding-challenges.md)

- [ ] [`/docs/russian/how-to-work-on-guide-articles.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/russian/how-to-work-on-guide-articles.md) from its english version at [`/docs/how-to-work-on-guide-articles.md`](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs/how-to-work-on-guide-articles.md)
 
#### Once you are ready please make a pull request, preferably one for each file.

### PR DESCRIPTION
**Pull request by olekspickle**
Translated all  overall descriptions and command descriptions for commands to work locally

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

Closes #18308

### CODE REVIEW

File: docs/russian/README.md [hunk 1]
```diff
@@ -11,41 +11,39 @@
...
 </table>
 
-# Documentation Quick Reference
+# Быстрая Справка По Документации
 
-Hello 👋 !
+Здравствуй 👋 !
 
-These instructions have not been translated yet. Please check this issue for details: [`#18312`](https://github.com/freeCodeCamp/freeCodeCamp/issues/18312)
+Эта папка содержит все описания по внесению вклада в freeCodeCamp.org
```
Comment (sergeyradov) [inline docs/russian/README.md]: " папка содержит" => "раздел содержит" ?

File: docs/russian/README.md [hunk 2]
```diff
@@ -11,41 +11,39 @@
...
 | ------- | ----------- |
-| `npm run bootstrap` | Bootstraps the different services |
-| `npm run seed` | Parse all the challenge markdown files and inserts them into MongoDB. |
-| `npm run develop` | Starts the freeCodeCamp API Server and Client Apps |
-| `npm test` |  Run all JS tests in the system, including client, server, lint and challenge tests |
-| `npm run test:client` | Run the client test suite |
-| `npm run test:curriculum` | Run the curriculum test suite |
-| `npm run test:server` | Run the server test suite |
-| `npm run commit` | An interactive tool to help you build a good commit message |
+| `npm run bootstrap` | Оборачивает разные сервисы |
```
Comment (sergeyradov) [inline docs/russian/README.md]: "Оборачивает" => "Подготавливает" Or "Запускает"

**Review by PurpleCatfish — CHANGES_REQUESTED** [review]

**Comment by sergeyradov** [comment]
@alexgnatrow Resolve conflict is needed. 
overall LGTM

**Review by greggubarev — APPROVED** [review]
**Thank you for opening this pull request.**

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. **We would welcome future pull requests from you.**

Thank you and happy coding.

**Pull request closed without merging**